ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/BottomToolbar/Menu/History/HistoryView.swift

```diff
@@ -15,134 +16,219 @@ import SwiftUI
-          .strokeBorder(Color(braveSystemName: .dividerSubtle), lineWidth: 1.0)
-      )
-      .containerShape(RoundedRectangle(cornerRadius: 6.0, style: .continuous))
-
-      VStack {
-        if let title = title, !title.isEmpty {
-          Text(title)
-            .font(.subheadline)
-            .lineLimit(2)
-            .truncationMode(.tail)
-            .frame(maxWidth: .infinity, alignment: .leading)
-            .fixedSize(horizontal: false, vertical: true)
-            .foregroundStyle(Color(braveSystemName: .textPrimary))
-        }
+    Label {
+      HStack(alignment: .top) {
+        VStack {
+          if let title = title, !title.isEmpty {
+            Text(title)
+              .lineLimit(1)
+              .truncationMode(.tail)
+              .frame(maxWidth: .infinity, alignment: .leading)
+              .fixedSize(horizontal: false, vertical: true)
+              .foregroundStyle(Color(braveSystemName: .textPrimary))
+          }
 
-        URLElidedText(
-          text: URLFormatter.formatURLOrigin(
-            forDisplayOmitSchemePathAndTrivialSubdomains: url.strippingBlobURLAuth.absoluteString
+          URLElidedText(
+            text: URLFormatter.formatURLOrigin(
+              forDisplayOmitSchemePathAndTrivialSubdomains: url.strippingBlobURLAuth.absoluteString
+            )
           )
+          .font(.footnote)
+          .frame(maxWidth: .infinity, alignment: .leading)
+          .fixedSize(horizontal: false, vertical: true)
+          .foregroundStyle(Color(braveSystemName: .textTertiary))
+        }
+
+        if let dateAdded {
+          Text(dateAdded, format: .dateTime.hour().minute())
+            .font(.caption2)
+            .foregroundStyle(Color(braveSystemName: .textTertiary))
+        }
+      }
+    } icon: {
+      FaviconImage(url: url, isPrivateBrowsing: false)
+        .frame(width: 20, height: 20)
+        .padding(6)
+        .overlay(
+          ContainerRelativeShape()
+            .strokeBorder(Color(braveSystemName: .dividerSubtle), lineWidth: 1.0)
         )
-        .font(.footnote)
-        .frame(maxWidth: .infinity, alignment: .leading)
-        .fixedSize(horizontal: false, vertical: true)
-        .foregroundStyle(Color(braveSystemName: .textSecondary))
+        .containerShape(RoundedRectangle(cornerRadius: 8.0, style: .continuous))
+    }
+  }
+}
+
+private struct RecentlyClosedTabsSection: View {
+  @FetchRequest(
+    entity: RecentlyClosed.entity(),
+    sortDescriptors: [.init(keyPath: \RecentlyClosed.dateAdded, ascending: false)]
+  ) private var recentlyClosedTabs: FetchedResults<RecentlyClosed>
+
+  @State private var isExpanded: Bool = false
+  var restoreTab: (RecentlyClosed) -> Void
+
+  var body: some View {
+    if !recentlyClosedTabs.isEmpty {
+      Section {
+        Button {
+          withAnimation {
+            isExpanded.toggle()
+          }
+        } label: {
+          Label {
+            HStack {
+              Text("Recently Closed Tabs")
```

Comment:
Localization

Reply:
fixed in https://github.com/brave/brave-core/pull/28816/commits/a6f7e738a4cffe7f7a8726fcdcdf20b10bc3a46e